src/types.js

```diff
@@ -2,8 +2,8 @@ import _ from 'lodash/fp'
 import * as F from 'futil-js'
 
 export let getTypeProp = _.curry((types, prop, node) =>
-  F.cascade([`${node.type}.${prop}`, `default.${prop}`], types)
+  F.cascade([`${node.type}.${prop}`, `default.${prop}`], types, () => true)
```

Comment:
The `() => true` probably should not be here (or should be an optional 4th param) because you wouldn't expect random properties to default to `() => true`. Keeping it on runTypeFunction is probably better